app/assets/javascripts/i18n.js

```diff
@@ -55,6 +55,11 @@
     return type === 'function' || type === 'object' && !!obj;
   };
 
+  var isFunction = function(func) {
+    var type = typeof func;
+    return type === 'function' && !!func;
```

Comment:
check if func exists, but it is probably not necessary here since `typeof` already block other values. Will remove the `!!func` check